packages/components/carousel/src/use-carousel.ts

```diff
@@ -494,7 +506,9 @@ export const useCarouselIndicators = () => {
       const isSelected = index === selectedIndex
 
       return {
```

Comment:
Is `aria-controls` not required?

Reply:
I had `aria-controls` set by `getControlProps`.

```ts
  const getControlProps: PropGetter<"button"> = useCallback(
    (props = {}, ref = null) => ({
      ...props,
      ref,
      "aria-controls": "carousel-items",
      disabled,
      onClick: handlerAll(props.onClick, onClick),
    }),
    [disabled, onClick],
  )
```

Does this mean that this is also a mistake?

Reply:
Refer to the end of the table below.

https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-1-prev-next/#rps_label

Reply:
No, I think we need to set it on the control button as well and have `aria-controls` on the indicator as well. I thought so.

Reply:
Thanks for letting me know! I will respond that way!